ingest_api/infrastructure/config.py

```diff
@@ -93,6 +94,11 @@ class IngestorConfig(BaseSettings):
         case_sensitive=False, env_file=".env", env_prefix="VEDA_", extra="ignore"
     )
 
+    git_sha: Optional[str] = Field(
+        subprocess.check_output(["git", "rev-parse", "HEAD"]).strip().decode("utf-8"),
```

Comment:
do we want to decode first (convert bytes to string) and then remove whitespace? https://docs.python.org/3/library/subprocess.html#subprocess.TimeoutExpired.output 
```suggestion
        subprocess.check_output(["git", "rev-parse", "HEAD"]).decode("utf-8").strip(),
```